File: app/src/main/java/com/example/mvvm/api/ApiCall.java
package com.example.mvvm.api;

import com.google.gson.JsonObject;

import java.util.Map;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;
import retrofit2.http.QueryMap;


public interface ApiCall {


    @GET("api")
    Call<JsonObject> login(@Query("username")String name,@Query("password")String asd);


}

File: app/src/main/java/com/example/mvvm/api/ApiCall.kt
package com.example.mvvm.api

import com.google.gson.JsonObject

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query
import retrofit2.http.QueryMap


interface ApiCall {


    @GET("api")
    fun login(@Query("username") name: String, @Query("password") asd: String): Call<JsonObject>


}
